.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Push Stable Package for Pypi
+
+on:
+  push:
+    branches:
+      - main
+      - "release/**"
```

Comment:
I thought that it would be better to release package to pypi when tags were pushed, what do you think?

Reply:
If we wanted to create release when commits were pushed on target branches, I would like to use automatic versioning like `MAJOR.MINOR.PATCH-rc+${date}.${commit_hash}` or something else, if possible.
Reason: I thought it would be hard for humans to update cfg every PR.

Reply:
I gave up on that due to some concerns, but will reexamine it and try to reimplement that policy.